bootstrap_cfn/stacks/iam-deploy.json

```diff
@@ -0,0 +1,74 @@
+{
+    "CrossAccountRole" : {
+        "Type" : "AWS::IAM::Role",
+        "Properties" : {
+            "AssumeRolePolicyDocument" : {
+                "Statement" : [
+                     {
+                         "Effect" : "Allow",
+                         "Principal" : {
+                             "AWS": "arn:aws:iam::ACCOUNT_NUMBER_HERE:root"
```

Comment:
Oh I see, this is munged by the config loader.

Rather than doing this and then changing the template inline how about we refer to a CFN Parameter. In the base.json:

```json
"Parameters" : {
  "DeployAccountID" : {
    "Type" : "Number",
    "Description" : "Put a useful desc here"
  }
}
```

Then in the stack this bit becomes:

```json
                         "Principal" : {
                             "AWS": { "Fn::Join" : ["",
                                 "arn:aws:iam::",
                                 { "Ref": "DeployAccountID" }
                                 ":root"
                             ] }
                         },
```

I think this makes it clearer - (it means that our python code has to do less at least)